src/commands/login/login.ts

```diff
@@ -3,8 +3,7 @@ import { OptionValues } from 'commander'
 import { chalk, exit, getToken, log } from '../../utils/command-helpers.js'
 import BaseCommand from '../base-command.js'
 
-// @ts-expect-error TS(7006) FIXME: Parameter 'location' implicitly has an 'any' type.
-const msg = function (location) {
+const msg = function (location: 'env' | 'flag' | 'config' | 'not found') {
```

Comment:
you can extract this type and move it into the types file as it's used later as well